.github/workflows/api_test.yml

```diff
@@ -43,21 +43,11 @@ jobs:
             | python3 - --version ${{ env.POETRY_VERSION }};
           poetry config virtualenvs.create true;
           poetry config virtualenvs.in-project true;
-      #----------------------------------------------
-      #       load cached venv if cache exists
-      #----------------------------------------------
-      - name: Load cached venv
-        id: cached-poetry-dependencies
-        uses: actions/cache@v4
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}-${{ matrix.python-version }}
-
+          
```

Comment:
I pulled the changes from https://github.com/Sage-Bionetworks/schematic/pull/1507 - As my build started to fail. I will pull in develop when this is merged in.